pkg/apis/eksctl.io/v1alpha4/types.go

```diff
@@ -445,4 +437,16 @@ type (
 		// +optional
 		ALBIngress *bool `json:"albIngress"`
 	}
+
+	// SSHConfig holds all the ssh access configuration to a NodeGroup
+	SSHConfig struct {
+		// +optional
+		Allow bool `json:"allow"`
```

Comment:
I think we should make this one `*bool` also, so that we can do meaningful defaulting for it :)